pkg/schedule/schedulers/scheduler_controller.go

```diff
@@ -486,7 +486,8 @@ retry:
 			if labelMgr == nil {
 				continue
 			}
-			if labelMgr.ScheduleDisabled(region) {
+			_, ok := s.Scheduler.(*evictLeaderScheduler)
```

Comment:
How about checking it before the loop? And please add some comments for this change.

Reply:
https://github.com/tikv/pd/pull/8303/commits/78200a44fac7f73a093af1974175fe9bb0071f05